helpdesk_mgmt/controllers/myaccount.py

```diff
@@ -259,3 +258,13 @@ def _ticket_get_order(self, order, groupby):
         if not field_name:
             return order
         return "%s, %s" % (field_name, order)
+
+    def _get_applicable_stages(self, team, close_from_portal=False):
+        domain = [
+            ("team_ids", "in", [False] + team.ids),
+        ]
+        if close_from_portal:
```

Comment:
Instead of passing this argument (which by the way the implementation is not correct, as it shouldn't be with OR, just filtered out in the proper place.

Reply:
And use only one method in one model. Don't put it twice.